README.md

```diff
@@ -30,24 +30,22 @@
 
 # Introduction
 
-The OSS Review Toolkit (ORT) assists with verifying Free and Open Source Software license compliance by checking a
-project's source code and its dependencies.
-
-From a bird's eye, it works by analyzing the project's build system for dependencies, downloading the source code of the
-dependencies, scanning all source code for license information, and summarizing the results.
-
-The different tools that make up ORT are designed as libraries (for programmatic use), with a minimal command line
-interface (for scripted use).
-
-The toolkit consists of the following tools:
-
-* [_Analyzer_](#analyzer) - determines dependencies of a project. Supports multiple package managers and sub-projects.
-  No changes to the projects are required.
-* [_Downloader_](#downloader) - fetches the source code referred to by the Analyzer result.
-* [_Scanner_](#scanner) - wraps existing license / copyright scanners to detect findings in local source code
-  directories.
-* [_Evaluator_](#evaluator) - evaluates license findings against customizable policy rules.
-* [_Reporter_](#reporter) - presents results in various formats such as visual reports, open source notices or
+The OSS Review Toolkit (ORT) aims to assist with the tasks that commonly need to be performed in the context of license
+compliance checks, especially for (but not limited to) Free and Open Source Software dependencies.
+
+It does so by orchestrating a _highly customizable_ pipeline of tools that _abstract away_ the underlying services.
+These tools are implemented as libraries (for programmatic use) and exposed via a command line interface (for scripted
+use):
+
+* [_Analyzer_](#analyzer) - determines the dependencies of projects and their meta-data, abstracting which package
+  managers or build systems are actually being used.
+* [_Downloader_](#downloader) - fetches all source code referred by the projects, abstracting which Version Control
```

Comment:
Maybe "all source code of the projects and their dependencies"?
"version control system"

Reply:
> Maybe "all source code of the projects and their dependencies"?

Done.

> "version control system"

You mean all lower case? I prefer to keep it capitalized like we already do in the "Downloader" section.

Reply:
Ok, but why do we do that, these are not proper nouns.

Reply:
I guess to emphasize the acronym being "VCS", which is why I also added it to make that clearer.